tests/stdlib/trandom.nim

```diff
@@ -187,3 +187,21 @@ block: # bug #17467
     doAssert x > 1e-4, $(x, i)
       # This used to fail for each i in 0..<26844, i.e. the 1st produced value
       # was predictable and < 1e-4, skewing distributions.
+
+when not defined(js) and not defined(macosx):
```

Comment:
how about remove the diff to tests/stdlib/trandom.nim to avoid conflict with https://github.com/nim-lang/Nim/pull/18149 ?

Reply:
sure, that makes more sense.